subtree-splits/blt-project/composer.json

```diff
@@ -33,9 +33,14 @@
     "extra": {
         "composer-exit-on-patch-failure": true,
         "drupal-scaffold": {
-            "initial": {
-                "sites/default/default.services.yml": "sites/default/services.yml",
-                "sites/default/default.settings.php": "sites/default/settings.php"
+            "locations": {
+                "web-root": "./docroot"
+            },
+            "allowed-packages": {
+                "drupal/core": "*"
+            },
+            "file-mapping": {
+                "[web-root]/sites/default/default.services.yml": false
```

Comment:
workaround for https://www.drupal.org/project/drupal/issues/3091285, make a note to remove this after it's fixed

Reply:
Workaround will be removed in DT-1076.